src/services/analytics/events/modals.ts

```diff
@@ -57,6 +57,16 @@ export const MODALS_EVENTS = {
     category: MODALS_CATEGORY,
     event: EventType.META,
   },
+  COPY_WARNING_SHOWN: {
```

Comment:
These events were meant for tx flow modals (not the best name).
I would move this to tx list events.